src/components/landingpage/UserChoicePanelRight.tsx

```diff
@@ -41,6 +41,11 @@ export function UserChoicePanelRight() {
       title: t('german'),
       langId: 'de',
     },
+    {
+      sno: 7,
+      title: t('portuguese/Brazil'),
```

Comment:
Punctuation should follow the same scheme as the other languages.
So "Portuguese" would be better.

Reply:
I have no idea - but would a Brazilian expect to see "Brazilian" or "Portugese"?

Please make a choice and capitalize the first letter so it's consistent in the UI.